### PR TITLE
kraken: manage headsign of trip in navitia

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.kisio.org/browse/NAVP-1070

### CODE REVIEW

File: source/kraken/tests/realtime_test.cpp [hunk 1]
```diff
@@ -2787,6 +2788,10 @@ BOOST_FIXTURE_TEST_CASE(add_and_update_trip_to_verify_route_line_commercial_mode
     BOOST_CHECK_EQUAL(res.response_type(), pbnavitia::ITINERARY_FOUND);
     BOOST_CHECK_EQUAL(res.journeys_size(), 1);
 
+    // Verify that headsign exists in display_informations
+    BOOST_CHECK_EQUAL(res.journeys(0).sections(0).pt_display_informations().headsign(), "trip_headsign");
+    BOOST_CHECK_EQUAL(res.journeys(0).sections(0).pt_display_informations().name(), "A - F");
```
Comment (pbougue) [inline source/kraken/tests/realtime_test.cpp:2793]: :+1: :eagle:

File: source/type/data.cpp [hunk 2]
```diff
@@ -62,7 +62,7 @@ namespace pt = boost::posix_time;
 namespace navitia {
 namespace type {
 
-const unsigned int Data::data_version = 69;  //< *INCREMENT* every time serialized data are modified
+const unsigned int Data::data_version = 70;  //< *INCREMENT* every time serialized data are modified
```
Comment (kinnou02) [inline source/type/data.cpp]: You do not need to increment `data_version` here, `Impacts` aren't in the `data.nav.lz4` as they are created by kraken at runtime.
Same case as: https://github.com/CanalTP/navitia/pull/2689#discussion_r262808530

**Review by pbougue — APPROVED** [review]
Agree with @kinnou02 on increment, otherwise it's alright!
> Makes me think that current "default" name is not always so great and uri has not always "vehicle_journey:" upfront... but it's another issue).

File: source/type/data.cpp (hunk 2 above)
Comment (kadhikari) [reply source/type/data.cpp]: Done

File: source/jormungandr/tests/kirin_realtime_tests.py [hunk 3]
```diff
@@ -1918,6 +1918,70 @@ def test_update_trip_with_physical_mode(self):
...
+        response = self.query_region(C_B_query)
+        assert has_the_disruption(response, 'new_trip')
+        self.is_valid_journey_response(response, C_B_query)
+        assert len(response['journeys']) == 2
+        pt_journey = response['journeys'][0]
+        assert 'non_pt_walking' not in pt_journey['tags']
+        assert pt_journey['status'] == 'ADDITIONAL_SERVICE'
+        assert pt_journey['sections'][0]['data_freshness'] == 'realtime'
+        assert pt_journey['sections'][0]['display_informations']['commercial_mode'] == 'additional service'
+        assert pt_journey['sections'][0]['display_informations']['headsign'] == 'trip_headsign'
```
Comment (nonifier) [inline source/jormungandr/tests/kirin_realtime_tests.py:1980]: Do you really need to test all the things other than the `headsign` ? 
It looks like copy/paste from another test and I don't really see the point on testing multiple time the same thing.

**Review by nonifier — APPROVED** [review]

File: source/jormungandr/tests/kirin_realtime_tests.py (hunk 3 above)
Comment (kadhikari) [reply source/jormungandr/tests/kirin_realtime_tests.py:1980]: You are right, i should have deleted some unnecessary lines

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**